lib/tizen_emulator.dart

```diff
@@ -326,10 +334,11 @@ class TizenEmulator extends Emulator {
         await _processUtils.run(<String>[emCli.path, 'launch', '--name', id]);
```

Comment:
> Also fixes a misleading output message when an emulator is not already running but fails to launch (exitCode == 2) for some reason.

I think this is because the emulator's CPU-VT option (Emulator Manager > Edit > HW Support) is set to OFF. (It seems that some emulators are created with OFF by default)

we can switch this option to ON with the following command
```
em-cli modify -n <emulator name> -w yes
```
How about adding something like the following before calling `emCli launch`?
```dart
// emulator's CPU-VT property must be set to ON to launch 
await _processUtils.run(<String>[emCli.path, 'modify', '--name', id, '-w', 'yes']);
```

Though, it is strange because the help message of `em-cli modify` says that the default option for -w is "yes". I  guess it is not properly applied.

Reply:
Resolved. There seems to be several issues that generate exit code 2. This would need to be investigated another time. The current code is sufficient.